network/containerizer/bridgepolicy.go

```diff
@@ -32,8 +32,10 @@ var skippedDeviceNames = set.NewStrings(
 // for guests inside of a host machine, along with the creation of network
 // devices on those bridges for the containers to use.
 type BridgePolicy struct {
-	// spaces is a cache of the model's spaces.
+	// spaces is a cache of the model's space IDs by name.
 	spaces map[string]string
+	// spaces is a cache of the model's space names by ID.
+	spaceNames map[string]string
```

Comment:
If this is a map of names by ID, what are we mapping to? Are we mapping to the name? Perhaps we should rename the other one to be `spaceIDs`?

Reply:
I clearly can't read properly. space ID -> space name is clearly mentioned.

Reply:
I renamed spaces to spaceIDs and updated the comments which both said spaces.